CHANGELOG.md

```diff
@@ -21,6 +21,12 @@ This project adheres to [Semantic Versioning](http://semver.org/).
    + Allows using or across related tables conditions
  - #1100, Customizable OpenAPI title - @AnthonyFisi
 
+### Fixed
+
+ - #2648, Fix inaccurate error codes with new ones - @laurenceisla
+   + `PGRST204`: Column is not found
+   + `PGRST003`: Timed out when acquiring connection to db
```

Comment:
This one could be a "fix", because https://github.com/PostgREST/postgrest/pull/2449 came out in 10.1.1